src/env.ts

```diff
@@ -7,7 +7,7 @@ export default validateEnv(
     ENV: Environment.DEVELOPMENT,
     GRAPHQL_ENDPOINT: 'http://0.0.0.0:8000',
     TRANSLATIONS_DOCUMENT: 'dcat_ap_no_validator_web',
-    ROUTE_BASE_NAME: '',
+    ROUTE_BASE_NAME: '/',
```

Comment:
I stedet for den, så er det et par ting som bør endres i webpack:
1) i base config, sett `publicPath: '/validator'`
2) i samme base config, sett htmlwebpackplugin base til `base: '/validator',`
3) i dev config, bruk `rewrites: [{ from: /./, to: '/validator/index.html' }]` under `historyApiFallback`
4) i prod config, ta bort `publicPath`, den blir da satt i `base.config.ts`